templates/accounts.go

```diff
@@ -146,6 +147,43 @@ transaction(publicKey: String) {
 }
 `
 
+const addAccountContractsTemplate = `
```

Comment:
Maybe rename this to "addOrUpdate", to indicate that this template may also update an existing contract (which is not safe in all cases!).

Maybe alternatively remove the update branch and add another separate template for it